daemons/ipa-kdb/ipa_kdb.c

```diff
@@ -524,6 +524,52 @@ static krb5_principal ipadb_create_local_tgs(krb5_context kcontext,
     return tgtp;
 }
 
+static char *no_attrs[] = {
+    LDAP_NO_ATTRS,
+
+    NULL
```

Comment:
style: why the blank line?

Reply:
I wrote it this way because it seems to be a common practice for LDAP attribute lists:
https://github.com/freeipa/freeipa/blob/release-4-10-1/daemons/ipa-kdb/ipa_kdb_pwdpolicy.c#L42
https://github.com/freeipa/freeipa/blob/release-4-10-1/daemons/ipa-kdb/ipa_kdb_principals.c#L97
https://github.com/freeipa/freeipa/blob/release-4-10-1/daemons/ipa-kdb/ipa_kdb_principals.c#L109

But this is not systematic, and given the fact the list has just one element, maybe it's a bit too pedantic to apply it here indeed.

Reply:
Ok, IMHO it's fine since it does stick with current style.